app/main.py

```diff
@@ -1,8 +1,62 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+
+        price = 0.0
+
+        for car in cars:
+
+            if car.clean_mark < self.clean_power:
+
+                car_income = self.calculate_washing_price(car)
+                price += car_income
+                car.clean_mark = self.wash_single_car()
```

Comment:
In the `serve_cars` method, the line `car.clean_mark = self.wash_single_car()` is misleading. The `wash_single_car` method returns `self.clean_power`, which is not necessarily the new clean mark of the car. Consider updating the car's clean mark based on a more realistic washing effect, or clarify the logic if this is intentional.